packages/components/src/responsive-wrapper/index.tsx

```diff
@@ -40,10 +38,6 @@ function ResponsiveWrapper( {
 	}
 
 	const TagName = isInline ? 'span' : 'div';
-	let aspectRatio;
-	if ( naturalWidth && naturalHeight ) {
```

Comment:
We should definitely not update this component(components package), but made me notice that we use that in PostFeaturedImage component. We might want to update there or remove this wrapper.